source/Command.cpp

```diff
@@ -85,7 +85,7 @@ const Command Command::HARVEST(ONE << 34, "Fleet: Harvest flotsam");
 const Command Command::AMMO(ONE << 35, "Fleet: Toggle ammo usage");
 const Command Command::AUTOSTEER(ONE << 36, "Auto steer");
 const Command Command::WAIT(ONE << 37, "");
-const Command Command::STOP(ONE << 38, "");
+const Command Command::STOP(ONE << 38, "Stop Ship");
```

Comment:
The command above all these constants states that the string is a description used in the control panel and as a key when saving/loading keys. That won't be true in this case, though, so perhaps we leave another comment here explaining that these three commands can't be set, and therefore a description is unneeded/unused (but that it is necessary for the mobile port).

Reply:
Well, it's the same situation as with fleet jump, but a comment wouldn't hurt.